packages/forms-web-app/src/dynamic-forms/dynamic-components/boolean/question.js

```diff
@@ -37,6 +37,36 @@ class BooleanQuestion extends RadioQuestion {
 			html
 		});
 	}
+
+	/**
+	 * returns the data to send to the DB
+	 * side effect: modifies journeyResponse with the new answers
+	 * @param {ExpressRequest} req
+	 * @param {JourneyResponse} journeyResponse - current journey response, modified with the new answers
+	 * @returns {Promise.<Object>}
+	 */
+	async getDataToSave(req, journeyResponse) {
```

Comment:
Wasn't expecting to see this file in this diff. How come this function didn't exist before?

Reply:
I'd guess mapping 'yes'|'no' to true/false.

Reply:
Exactly.  The superclass Question has a getDataToSave function which (broadly) extracts the value input on the relevant question page, sets it to be saved and updates the journeyResponse object.  BooleanQuestions return strings from the front end (I think it's a nunjucks thing) so we need to override the default getDataToSave and convert to a boolean.  At one stage I had the logic in the main question.js file, but thought it made more sense to extract it out to different types where the corresponding field in the schema doesn't match the value returned from the front end